src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/evaluators/PVariantEvaluator.java

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.hellbender.tools.walkers.varianteval.evaluators;
+
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.walkers.varianteval.util.Analysis;
+import org.broadinstitute.hellbender.tools.walkers.varianteval.util.DataPoint;
+
+@Analysis(description = "Computes different estimates of theta based on variant sites and genotypes")
```

Comment:
What's theta?

Reply:
I think it'd be good to document the method succinctly; i.e. divide sites by AF in 1000G, then within each bin add up the alt alleles and divide by 2*num_sites, and do the goodness of fit test to get the p-value.

Reply:
I think that documentation should be for `AllleFrequencyQC`; this is just the evaluator being used to calculate average var allele fraction (P --> why its PVariantEvaluator) and count the number of hets/hom vars/ hom refs; definitely updating description but not sure about a new name unless you have a better idea ?

Reply:
Yea, docs should go wherever you think is appropriate, I just happened to think that when I was reviewing this part.

As for the name, maybe something like AggregateAltFraction

Reply:
How about VariantAFEvaluator ?
I think there is decent documentation in `AlleleFrequencyQC.java` -- let me know if you think it's unclear?

Reply:
I think the first sentence in `AlleleFreqeuncyQC.java` docs "This tool uses Variant Eval..." could be improved. What do you think?